packages/app/src/pages/SwapPage/SwapComponent.tsx

```diff
@@ -122,7 +128,7 @@ export function SwapComponent({
           {...fromInput.getInputProps()}
           {...(activeInput.current === ActiveInput.to && { isLoading })}
           autoFocus={activeInput.current === ActiveInput.from}
-          coinSelectorDisabled={true}
+          coinSelectorDisabled={coinFrom?.symbol === "ETH"}
```

Comment:
You can use CoinETH const form `~/lib/constants` to check here.

Reply:
✅